centml/cli/cluster.py

```diff
@@ -158,82 +174,54 @@ def get(type, id):
         )
 
 
-# Define common deployment
-def common_options(func):
+# fetching hardware pricing listing
+def load_hw_choices():
```

Comment:
is there any reason to have a separate function for this? Why not just do this inside the __init__() of HardwarePricingTier?

Reply:
moved the logic to inside init method

Reply:
I think we can get rid of this function all together.